libyaml_q_declaration.md

```diff
@@ -85,7 +89,7 @@ Provided [doxygen documentation](https://github.com/yaml/libyaml/tree/master/doc
 
 ### Public API Documentation [3.ii]
 
-Yes, doxygen documentation is available for library .h headers.
+Yes, doxygen documentation is available for library [.h headers]([yaml.h](https://github.com/yaml/libyaml/blob/master/include/).
```

Comment:
This link is malformed.

How about linking to their rendered documentation?

https://libyaml.docsforge.com/master/